learning-objectives/intl/es.yml

```
@@ -1204,18 +1210,72 @@ typescript/classes/members:
     - title: Documentación oficial de Typescript
       url: https://www.typescriptlang.org/docs/handbook/2/classes.html#class-members
 
+typescript/classes/members/fields:
+  title: Campos
+  links:
+    - title: Documentación oficial de Typescript
+      url: https://www.typescriptlang.org/docs/handbook/2/classes.html#fields
+
+typescript/classes/members/constructors:
+  title: Constructors
+  links:
+    - title: Documentación oficial de Typescript
+      url: https://www.typescriptlang.org/docs/handbook/2/classes.htmll#constructors
```

Esta URL arroja error `404`, creo que arreglando esa, estaría OK el PR para ser mergeado:

```suggestion
      url: https://www.typescriptlang.org/docs/handbook/2/classes.html#constructors
```